src/components/colorPicker/index.tsx

```diff
@@ -70,6 +71,14 @@ class ColorPicker extends PureComponent<Props> {
     backgroundColor: Colors.$backgroundDefault
   };
 
+  constructor(props: Props) {
+    super(props);
+
+    if (props.style) {
+      LogService.warn(`UILib ColorPicker style prop is deprecated, please pass instead the 'backgroundColor' prop.`);
```

Comment:
'backgroundColor' prop is missing from the api.json...

Reply:
I notice that the 'backgroundColor' prop is missing in the ColorPalette api.json too.
I'll do it in another PR.